cms/search/wagtail_hooks.py

```diff
@@ -0,0 +1,41 @@
+from typing import TYPE_CHECKING
+
+from wagtail import hooks
+from wagtail.models import Page
+
+from . import get_publisher
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+
+publisher = get_publisher()
+
+
+def _should_exclude_page(page: Page) -> bool:
+    """Return True if the page should be excluded from publishing."""
+    return page.__class__.__name__ in EXCLUDED_PAGE_TYPES
+
+
+@hooks.register("after_publish_page")
+def page_published(request: "HttpRequest", page: Page) -> None:
+    """This hook is called after a page is published from the Wagtail admin."""
```

Comment:
Let's clarify this doc string. Might be Admin UI or programmatic. Not sure we need the doc string. Could link to doc source.

Reply:
Okay, understood. Done!